hosted/eks/helper/helper_cluster.go

```diff
@@ -540,9 +540,27 @@ func DeleteEKSClusterOnAWS(eks_region string, clusterName string) error {
 	}()
 	_ = os.Setenv("KUBECONFIG", downstreamKubeconfig)
 
+	fmt.Println("Deleting all the nodegroups ...")
+	ngNames, err := GetFromEKS(region, clusterName, "nodegroup", ".[].Name")
+	if err != nil {
+		return errors.Wrap(err, "Failed to list nodegroup for deletion")
+	}
+
+	if ngNames != "" {
+		for _, ngName := range strings.Split(ngNames, "\n") {
+			args := []string{"delete", "nodegroup", "--region", region, "--name", ngName, "--cluster", clusterName, "--disable-eviction", "--wait"}
```

Comment:
nit:
Could we use _ModifyEKSNodegroupOnAWS_ here ?

Reply:
Oh yes, of course. I was going to use that, and then forgot :sweat_smile: